regparser/web/jobs/utils.py

```diff
@@ -88,4 +129,4 @@ def status_url(job_id):
     hostport = getattr(settings, "CANONICAL_PORT", "")
     if hostport:
         hostport = ":%s" % hostport
-    return "%s%s/rp/job/%s/" % (hostname, hostport, job_id)
+    return "%s%s/rp/job/upload/%s/" % (hostname, hostport, file_hash)
```

Comment:
There's quite a bit of redundancy between this function and `status_url`. Obviously not a blocker, but I'd encourage you to think through ways to remove the repetition.